packages/central-server/src/apiV2/utilities/permissionsBasedChangeFilter.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { SqlQuery } from '@tupaia/database';
+import semverCompare from 'semver-compare';
+
+export const PERMISSIONS_BASED_SYNC_MIN_APP_VERSION = '1.12.123';
+
+export const supportsPermissionsBasedSync = version =>
+  semverCompare(version, PERMISSIONS_BASED_SYNC_MIN_APP_VERSION) >= 0;
+
+export const getCountriesAndPermissionsToSync = async req => {
```

Comment:
Here's where we do the little shimmy to work out which new countries and permission groups we need to sync. Will add more comments around this code explaining why